statistics/handle/lockstats/unlock_stats.go

```diff
@@ -55,14 +57,31 @@ func RemoveLockedTables(exec sqlexec.RestrictedSQLExecutor, tids []int64, pids [
 	if err != nil {
 		return "", err
 	}
-	skippedTables := make([]string, 0, len(tables))
+	skippedTables := make([]string, 0, len(tidAndNames))
+	tids := make([]int64, 0, len(tidAndNames))
+	tables := make([]string, 0, len(tidAndNames))
+	for tid, tableName := range tidAndNames {
+		tids = append(tids, tid)
+		tables = append(tables, tableName)
+	}
```

Comment:
There is a small side effect of the change.
Because the order of iterating a `map` is not deterministic in Go, the content in `tids`, therefore `skippedTables`, and finally the returned `msg` would not be stable.
And it may make some tests unstable. I think the current `TestRemoveLockedTables` is affected by this.

Reply:
I think we can just modify the test case to work around this :laughing:

Reply:
Good catch! 👍🏿

Reply:
I think maybe we can fix that to stabilize the performance. I guess it wouldn't be a big deal in terms of performance.